plans/artifactory-pro/hooks/run

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+echo "Starting Artifactory Pro"
+
+export JAVA_HOME=$(hab pkg path core/server-jre)
+export ARTIFACTORY_HOME={{pkg.svc_var_path}}
+
+{{pkg.svc_var_path}}/bin/artifactory.sh 2>&1
```

Comment:
So like `pkg_svc_run="JAVA_HOME=$(pkg_path_for core/server-jre) ARTIFACTORY_HOME=$pkg_svc_var_path pkg_svc_var_path/bin/artifactory.sh 2>&1" wouldn't work? (I think `pkg_path_for` should pull it??)